### PR TITLE
compatibilty 3.3 <-> 3.4

### PR DESCRIPTION
**Pull request by ObiWahn**
Please note that for legal reasons we require you to sign the 
[Contributor Agreement](https://www.arangodb.com/documents/cla.pdf)
before we can accept your pull requests.

### CODE REVIEW

**Comment by jsteemann** [comment]
Please remove the following sentence from `Documentation/Books/Manual/ReleaseNotes/UpgradingChanges34.md`:

> the `NEAR` AQL function now does not default to a limit of 100 documents
> any more, but will return all documents if no limit is specified.

And please add tests for the changed behavior and attributes, so that we are sure it works.

File: Documentation/Books/Manual/ReleaseNotes/UpgradingChanges34.md [hunk 1]
```diff
@@ -190,9 +190,11 @@ APIs:
   AQL user functions on the top level of the response.
   Each AQL user function description now also contains the 'isDeterministic' attribute.
 
-- `GET /_admin/status` now returns the attribute `operationMode` instead of `mode`.
-  The previously existing attribute `writeOpsEnabled` is no longer returned and was
-  replaced with an attribute `readOnly` with the inverted meaning.
+- `GET /_admin/status` now returns the attribute `operationMode` in addition to
+  `mode`. The attribute `writeOpsEnabled` is now also represented by the new an
+  attribute `readOnly`, which is has an inverted value compared to the original
+  attribute. In future releases to old attributes will be deprecated in favour
```
Comment (jsteemann) [inline Documentation/Books/Manual/ReleaseNotes/UpgradingChanges34.md]: "to old" => "the old"
"favour" => "favor" (BE => AE)

File: CHANGELOG [hunk 2]
```diff
@@ -97,9 +97,9 @@ devel
   The `padded` key generator generates keys of a fixed length (16 bytes) in
   ascending lexicographical sort order.
 
-* The REST API of `/_admin/status` changed: Field "mode" is now named "operationMode",
-  field "writeOpsEnabled" no longer exists and is replaced by field "readOnly" with
-  the inverted meaning.
+* The REST API of `/_admin/status` added: "operationMode" filed with same meaning as
+  the "mode" field and field "readOnly" that has the inverted meaning of field
```
Comment (dothebart) [inline CHANGELOG]: of THE field

**Comment by jsteemann** [comment]
http://jenkins01.arangodb.biz:8080/view/PR/job/arangodb-matrix-pr/413/

**Pull request merged**